src/styles/blocks/snake.scss

```
@@ -0,0 +1,9 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  @media (min-width: 1200px) {
+    & {
+      grid-template-columns: 1fr 1fr 1fr;
```

```suggestion
      grid-template-columns: repeat(3 ,1fr);
```
так як в тебе вище є `repeat` - тут можна так само